### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by fpsapc**
- Use external API: [API Ninjas](https://api-ninjas.com/api/quotes)
- Create a new component named Quote
- Data fetched inside the new component
- A loading state showed
- An error state showed
- The new component Imported into your main component (next to the Calculator)
- It is on the same page as the Calculator

### CODE REVIEW

File: src/components/Quote.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        const result = await response.json();
+        setQuote(result[0]);
+        setLoading(false);
+      } catch (error) {
+        setError(error.message);
+        setLoading(false);
+      }
+    };
+    fetchQuote();
+  }, []);
```
Comment (AmaduKamara) [inline src/components/Quote.js]: - [x] Good job on your implementation. Things are working well but it's highly considered to pass in dependencies to the dependency array of the useEffect hook. Any variable used in the hook that is not created inside it should be passed as a dependency array. Kindly add them 👍🏽

```javascript
 fetchQuote();
  }, [setQuote, setLoading]);
```

**Review by AmaduKamara — APPROVED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi @fpsapc,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed successfully 
✅ The app works fine 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add the dependencies to the dependency array of the useEffect for the proper use of the hook 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Quote.js (hunk 1 above)
Comment (fpsapc) [reply src/components/Quote.js]: @AmaduKamara Thanks for your review

**Review by sumairq — APPROVED** [review]
### Hi @fpsapc , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**